lesson7_task2.py

```diff
@@ -0,0 +1,41 @@
+"""
+Input data:
+stock = {
+    "banana": 6,
+    "apple": 0,
+    "orange": 32,
+    "pear": 15
+}
+prices = {
+    "banana": 4,
+    "apple": 2,
+    "orange": 1.5,
+    "pear": 3
+}
+Create a function which takes as input two dicts with structure mentioned above, then computes and returns
+the total price of stock.
+"""
+
+
+def total_price_of_stock(stock: dict, prices: dict) -> int | float:
```

Comment:
Nice solution, appreciate your typehints, you may even enhance them, like: dict[str, int | float], not just dict